app/controllers/concerns/api_csv.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module ApiCsv
+  extend ActiveSupport::Concern
```

Comment:
Don't need to extend Concern unless you're using `included` `prepended` or one of the other methods that it adds. This is just a plain ol' grouping module.

Reply:
Good spot :)